### PR TITLE
Add update available property

### PR DESCRIPTION
**Pull request by starkillerOG**

### CODE REVIEW

**Comment by starkillerOG** [comment]
I would like to add a check in HomeAssistant to see if their are updates available for the hue/bridge or lights.
I could then issue a logger warning prompting the user to update the bridge/bulb if their is an update available.

This is related to the color gamut, as that requires up to date firmware to function properly.

File: aiohue/lights.py [hunk 1]
```diff
@@ -61,6 +61,11 @@ def swversion(self):
         """Software version of the light."""
         return self.raw['swversion']
 
+    @property
+    def swupdateavailable(self):
+        """Software update available for the light."""
+        return self.raw.get('swupdate', {}).get('state', 'not updatable')
```
Comment (balloob) [inline aiohue/lights.py]: I don't think that we should fill in `not updatable` when there is no value. Instead just let it return `None`

Comment (starkillerOG) [reply aiohue/lights.py]: Alright, will change it

File: aiohue/lights.py [hunk 2]
```diff
@@ -61,6 +61,11 @@ def swversion(self):
         """Software version of the light."""
         return self.raw['swversion']
 
+    @property
+    def swupdateavailable(self):
```
Comment (balloob) [inline aiohue/lights.py]: can we rename this to `swupdatestate`, to reflect the property names in the API?

Comment (starkillerOG) [reply aiohue/lights.py]: just renamed it

**Comment by balloob** [comment]
Released as 1.9.0

**Pull request merged**